src/classes/path-utils.ts

```diff
@@ -11,36 +11,36 @@ const defaultProjectPath = "~/.meta-proj-cli/projects";
 /**
  * Offers functions that help with paths
  * 
- * fixPath : changes file separators to platform specific ones and expands tilde(~) paths
- * outerFolder : gives the path format to the outer folder of a project
- * repoFolder : gives the path format to the folder that actually holds the project (contains .git)
+ * fixPath: changes file separators to platform specific ones and expands tilde(~) paths
+ * outerFolder: gives the path format to the outer folder of a project
+ * repoFolder: gives the path format to the folder that actually holds the project (contains .git)
```

Comment:
These descriptions should be on function docs not in class docs

Reply:
Moved tsdocs, added tsdocs